Tests/LinuxMain.swift

```diff
@@ -72,6 +72,14 @@ extension StatisticsTests {
     ]
 }
 
+extension TemplateCheckerTests {
+    static var allTests: [(String, (TemplateCheckerTests) -> () throws -> Void)] = [
+        ("testPerformWithLocalSource", testPerformWithLocalSource),
+        ("testPerformWithRemoteSource", testPerformWithRemoteSource),
+        ("testPerformWithGithubSource", testPerformWithGithubSource)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)